SharedPackages/BrowserServicesKit/Tests/BrowserServicesKitTests/Subscription/SubscriptionFeatureAvailabilityTests.swift

```diff
@@ -130,6 +130,13 @@ final class SubscriptionFeatureAvailabilityTests: XCTestCase {
 }
 
 class MockPrivacyConfiguration: PrivacyConfiguration {
+    func isSubfeatureEnabled(_ subfeature: any BrowserServicesKit.PrivacySubfeature, versionProvider: BrowserServicesKit.AppVersionProvider, randomizer: (Range<Double>) -> Double, defaultValue: Bool) -> Bool {
+        isSubfeatureEnabledCheck?(subfeature) ?? false
```

Comment:
A bit of a nitpick :) I know these are just mocks and you mimic the existing implementation, but the good practice would be to call the original function - or - the other way around, call this from the original one. Generally - to keep one code path so if it needs to be changed, we do it in one place.